app/src/main/java/io/intrepid/animationexercise/screens/detail/DetailActivity.java

```diff
@@ -101,6 +118,16 @@ public void showCat() {
     public void animatePowerUp() {
         Drawable drawable = powerUpBar.getDrawable();
         drawable.setLevel(10000);
+        ObjectAnimator moveAnim = ObjectAnimator.ofFloat(powerUpBar, "Y", 1000);
```

Comment:
You want to animate the property "level" instead of "Y", and it goes all the way to 10000, not 1000.